types/skuid.go

```diff
@@ -13,7 +13,7 @@ import (
 	"path/filepath"
 )
 
-type PullResponse struct {
+type PullResponse struct { 
```

Comment:
`go fmt` should clean this up for you, can you run `go fmt ./...`  on the project?